### PR TITLE
Master hello

### PR DESCRIPTION
**Pull request by rocketnik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1013,7 +1013,6 @@ Let us know about it, we would be glad to mention it here. :)
 Specify Eureka into your project's `Podfile`:
 
 ```ruby
-source 'https://github.com/CocoaPods/Specs.git'
```
Comment (Nonnus) [inline README.md]: should this really be removed?

File: Source/Core/Helpers.swift [hunk 2]
```diff
@@ -67,10 +67,11 @@ extension NSExpression {
         switch expressionType {
             case .function, .variable:
                 let str = "\(self)"
-                if let range = str.range(of: ".") {
-                    return [String(str[str.index(str.startIndex, offsetBy: 1)..<range.lowerBound])]
+                let substring = str[str.index(str.startIndex, offsetBy: 1)...]
```
Comment (Nonnus) [inline Source/Core/Helpers.swift:70]: nice, much cleaner logic

Comment (rocketnik) [reply Source/Core/Helpers.swift:70]: Sombody else of our team did this.

File: README.md (hunk 1 above)
Comment (rocketnik) [reply README.md]: i was also wondering about it. I added it back.

**Review by Nonnus — APPROVED** [review]
code looks good, will jusr checkout and run on my machine to test

**Pull request merged**